ntoskrnl/ke/bug.c

```diff
@@ -1080,14 +1080,7 @@ KeBugCheckWithTf(IN ULONG BugCheckCode,
         KeBugCheckOwner = Prcb->Number;
 
         /* Freeze the other CPUs */
-        for (ULONG i = 0; i < KeNumberProcessors; i++)
-        {
-            if (i != Prcb->Number)
-            {
-                /* Send the IPI */
-                KiIpiSend(AFFINITY_MASK(i), IPI_FREEZE);
-            }
-        }
```

Comment:
Side-note: An alternative to the KxFreeze would have been to call
```c
KiIpiSend(KeActiveProcessors & ~Prcb->SetMember, IPI_FREEZE);
```
like what's done inside KxFreezeExecution(), but without the extra stuff that would allow them to be revived later.

Reply:
No, that wouldn't work. Freezing requires IpiFrozen to be set appropriately for all processors.